### ISSUE TITLE
browser.getDevices() doesn't find my Airport Express

### ISSUE THREAD

**Issue by jeena** [benvanik/node-airplay#1]
Hi,

I have bought a Airport Express to be able to listen to music from my computer in my living room. So naturally I wanted to play a bit with it too and I found this project. I installed the airplay module and tried the example code but even if I can play music on the Airport Express from iTunes, node-airplay doesn't discover my Airport Express:

```
> var browser = require('airplay').createBrowser();
undefined
> console.log(browser.getDevices());
[]
undefined
```

I just get an empty array. What am I doing wrong?

**Comment by lemieuxster** [comment]
It seems that airport express uses AirTunes which uses `raop` not `airplay` (browser.js, line 13) - RTSP (the protocol) is different than HTTP and would require quite a few changes, I imagine, to make this script interoperable.

**Comment by benvanik** [comment]
Yeah it's unfortunate that they use totally different protocols. It'd be fun to have a node-raop project (if one doesn't exist), then any project looking to use this could abstract out the protocol (and also add upnp/etc).
